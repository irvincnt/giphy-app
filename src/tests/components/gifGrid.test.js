
import React from 'react';
import { shallow } from "enzyme"
import GifItem from '../../components/gifItem';

describe('Pruebas en <gifGrid>', () => {
  test('Debe mostrar el componente correctamente', () => {
    const wrapper = shallow(<GifItem />)

    expect(wrapper).toMatchSnapshot();
  })
})