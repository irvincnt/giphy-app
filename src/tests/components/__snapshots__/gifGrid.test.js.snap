// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <gifGrid> Debe mostrar el componente correctamente 1`] = `
<div
  className="card-item"
>
  <img />
  <p />
</div>
`;
